### DISCUSSION TITLE
Private repo?

### DISCUSSION THREAD

**Discussion by jbhrfx9280** [nickjj/docker-django-example#27] [Q&A]
It appears that docker-django-example is in a public forum but to implement this example you must have a paid subscription to Docker. "python:3.10.5-slim-bullseye" (Dockerfile) is in a private repo. There are no public python 3.10.5 reliable images available. Has this current example been tested with python 3.9?

**Comment by nickjj** [comment]
The `python` image that this project uses is publicly available on `https://hub.docker.com/_/python`. You don't need to pay anything to access it. It's public and free. It's the official Python image from Docker.

That specific tag can be found here: https://hub.docker.com/_/python/tags?page=1&name=3.10.5
